CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/GalleryFragment.kt

```diff
@@ -111,8 +112,8 @@ class GalleryFragment internal constructor() : Fragment() {
                     val mediaType = MimeTypeMap.getSingleton()
                             .getMimeTypeFromExtension(mediaFile.extension)
                     // Get URI from our FileProvider implementation
-                    val uri = FileProvider.getUriForFile(
-                            view.context, BuildConfig.APPLICATION_ID + ".provider", mediaFile)
+                    val uri = FileProvider.getUriForFile(view.context,
+                            BuildConfig.APPLICATION_ID + ".provider", mediaFile)
```

Comment:
Please change this back to the previous form, which is more appropriate as per Kotlin style guides. My interpretation is that you should either put all arguments of a function into a single line (same line as the function name, or the next line) otherwise each argument should be in its own line.